WordPress/Classes/ViewRelated/Stats/Extensions/WPStyleGuide+Stats.swift

```diff
@@ -111,11 +111,17 @@ extension WPStyleGuide {
             return UIImage(named: "gravatar")
         }
 
-        static func configureFilterTabBar(_ filterTabBar: FilterTabBar) {
-            filterTabBar.tintColor = filterTintColor
+        static func configureFilterTabBar(_ filterTabBar: FilterTabBar, forTabbedCard: Bool = false) {
+            filterTabBar.dividerColor =  filterDividerColor
             filterTabBar.deselectedTabColor = filterDeselectedColor
-            filterTabBar.dividerColor = filterDividerColor
-            filterTabBar.tabSizingStyle = .equalWidths
+            filterTabBar.tintColor = defaultFilterTintColor
+
+            // For FilterTabBar on TabbedTotalsCell
+            if forTabbedCard {
+                filterTabBar.tabSizingStyle = .equalWidths
+                filterTabBar.tintColor = tabbedCardFilterTintColor
+                filterTabBar.selectedTitleColor = defaultTextColor
```

Comment:
Is it worth defining another `static let` color for the title color instead of reusing `defaultTextColor`?

Reply:
Oops, I missed that one. Tis done! Thanks @frosty !